src/producer.rs

```diff
@@ -143,7 +143,7 @@ impl Archive {
             FilePath::Path(path) => match File::open(path) {
                 Ok(mut f) => checker(&mut f),
                 Err(_) => false,
-            },
+            }
```

Comment:
Nit: could you bring back the comma to avoid affecting 'blame' information

Reply:
Of course, thanks for your patience.

Reply:
No worries, that's really minor!